xmake/modules/package/manager/cmake/find_package.lua

```diff
@@ -78,19 +78,20 @@ function _find_package(cmake, name, opt)
         end
     end
     local testname = "test_" .. name
+    local findname = configs.find_name or name
     cmakefile:print("find_package(%s REQUIRED %s)", requirestr, componentstr)
     cmakefile:print("if(%s_FOUND)", name)
     cmakefile:print("   add_executable(%s test.cpp)", testname)
     cmakefile:print("   target_include_directories(%s PRIVATE ${%s_INCLUDE_DIR} ${%s_INCLUDE_DIRS})",
-        testname, name, name)
+        testname, findname, findname)
     cmakefile:print("   target_include_directories(%s PRIVATE ${%s_INCLUDE_DIR} ${%s_INCLUDE_DIRS})",
-        testname, name:upper(), name:upper())
+        testname, findname:upper(), findname:upper())
```

Comment:
I think we should keep `configs.find_name` as is, instead of calling upper() on it, some packages may have lowercase find_name.

Reply:
we actually check both findname (line 86) and findname:upper() (line 88), this is for `findname = name` but the library have all uppercase variables.